src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.Notifications.CustomScript
+{
+    public class CustomScript : NotificationBase<CustomScriptSettings>
```

Comment:
ExternalScript vs CustomScript isn't that different imho.
ExternalScript makes it more clear that it's outside of Sonarr.
CustomScript makes it more clear that it's a user-created script.

But anyone not understanding what it means shouldn't fiddle with it anyway.

I honestly dunno what you should pick. So lets vote for the least work :wink: